### PR TITLE
update build cloud image docs

### PR DESCRIPTION
**Pull request by willzhang**
Signed-off-by: willzhang <willzhmic@outlook.com>

### CODE REVIEW

File: docs/4.0/docs/build-cloud-image/build-cloudImage-with-binary-file.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Build cloud image with binary file
 
-Command line tools such as helm or kustomize are single binary files. We can package them into cloud images, and then install them by deploying cloud images on the master node.Let's take helm as an example to introduce how to package binary files into cloud image.
+Command line tools such as helm or kustomize are single binary files. Sealos can package them into cloud images, and then install them by deploying cloud images on the master node.Let's take helm as an example to introduce how to package binary files into cloud image.
```
Comment (fanux) [inline docs/4.0/docs/build-cloud-image/build-cloudImage-with-binary-file.md:3]: Sealos->sealos

File: docs/4.0/docs/build-cloud-image/build-cloudImage-with-binary-file.md [hunk 2]
```diff
@@ -24,30 +24,23 @@ chmod a+x linux-amd64/helm
...
-    ├── charts
-    ├── Chart.yaml
-    ├── README.md
-    ├── templates
-    ├── values.schema.json
-    └── values.yaml
-```
-
-Create a Kubefile for image build:
+Create a file named `Kubefile` for image build:
```
Comment (fanux) [inline docs/4.0/docs/build-cloud-image/build-cloudImage-with-binary-file.md:27]: Rename Kubefile to `Sealosfile`?

File: docs/4.0/docs/build-cloud-image/build-cloudImage-with-helm-charts.md [hunk 3]
```diff
@@ -152,4 +153,80 @@ If you use a private image repository, just login registry with `sealos login` b
...
+          image: "grafana/loki:2.0.0"
+          image: "bats/bats:v1.1.0"
+          image: bats/bats:v1.1.0
+```
+
+
+
+
+
+
```
Comment (fanux) [inline docs/4.0/docs/build-cloud-image/build-cloudImage-with-helm-charts.md:227]: extra spaces

File: docs/4.0/docs/build-cloud-image/build-cloudImage-with-binary-file.md (hunk 1 above)
Comment (fengxsong) [reply docs/4.0/docs/build-cloud-image/build-cloudImage-with-binary-file.md:3]: Command line tools such as helm and kustomize are single binary files. sealos can pack them into a cloud image, and then install those binaries by deploying cloud images on the master node. Let's take helm as an example to show you how to package binaries into cloud image.

File: docs/4.0/docs/build-cloud-image/build-cloudImage-with-helm-charts.md [hunk 4]
```diff
@@ -152,4 +153,80 @@ If you use a private image repository, just login registry with `sealos login` b
...
+$ cat charts/loki-stack.values.yaml
+promtail:
+  enabled: false
+fluent-bit:
+  enabled: true
+grafana:
+  enabled: true
+```
+
+Different values.yaml files will parses different images, which is the purpose of customizing values.yml - to enable sealos to automatically parses the images during run `sealos build` .
```
Comment (fengxsong) [inline docs/4.0/docs/build-cloud-image/build-cloudImage-with-helm-charts.md:206]: Different values files may be output to a different list of images

to enable sealos to automatically parses the images during `sealos build`.

**Review by willzhang — COMMENTED** [review]
update

**Pull request merged**